### PR TITLE
Feat/unit test

### PR DESCRIPTION
**Pull request by rasheedsulayman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Android Commons
 A library for faster development of android applications by Cotta & Cush Ltd.
 
+[![Build Status](https://travis-ci.org/CottaCush/AndroidCommons.svg?branch=feat%2Funit_test)](https://travis-ci.org/CottaCush/AndroidCommons)
```
Comment (goke-epapa) [inline README.md]: Change branch to master

File: README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+[![codecov](https://codecov.io/gh/CottaCush/AndroidCommons/branch/feat%2Funit_test/graph/badge.svg)](https://codecov.io/gh/CottaCush/AndroidCommons)
```
Comment (goke-epapa) [inline README.md]: Change branch to `master`

File: app/src/androidTest/java/com/cottacush/android/libraries/ExampleInstrumentedTest.java [hunk 3]
```diff
@@ -16,11 +48,16 @@
...
 public class ExampleInstrumentedTest {
+    private final String prefName = "TEST_PREF";
+    private  Context context;
+    @Before
+    public void setUp() throws Exception {
+        context = InstrumentationRegistry.getContext();
+    }
+
     @Test
     public void useAppContext() throws Exception {
```
Comment (goke-epapa) [inline app/src/androidTest/java/com/cottacush/android/libraries/ExampleInstrumentedTest.java]: This method is redundant now

Comment (goke-epapa) [reply app/src/androidTest/java/com/cottacush/android/libraries/ExampleInstrumentedTest.java]: You can assert that context is not null here

**Review by goke-epapa — CHANGES_REQUESTED** [review]
Try to test for both failure case and success case.

File: README.md (hunk 1 above)
Comment (rasheedsulayman) [reply README.md]: The report label will break if i change it to masters(there are no reports for masters branch yet) . I think we should merge first , the update the masters directly to reflect the changes.

**Review by goke-epapa — APPROVED** [review]

**Pull request merged**